### PR TITLE
Add react starter basic webpack

### PR DESCRIPTION
**Pull request by abechoe**
This PR adds some basic webpack configuration, along with boilerplate react code

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,5 +2,17 @@
...
   "main": "index.js",
-  "license": "MIT"
+  "license": "MIT",
+  "devDependencies": {
+    "babel-core": "^6.26.3",
+    "babel-loader": "^7.1.5",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.24.1",
+    "webpack": "^4.16.5",
+    "webpack-cli": "^3.1.0"
```
Comment (yirmiyahu) [inline package.json]: There might be newer versions for these guys.

Comment (yirmiyahu) [reply package.json]: I'd also try to get webpack dev server running.

Comment (abechoe) [reply package.json]: For webpack, what I have here are the latest versions:
https://github.com/webpack/webpack/releases
https://github.com/webpack/webpack-cli/releases

Comment (yirmiyahu) [reply package.json]: I was considering also the `babel`s.

Comment (abechoe) [reply package.json]: Are you referring to https://www.npmjs.com/package/@babel/preset-env and https://www.npmjs.com/package/@babel/preset-react?

Comment (yirmiyahu) [reply package.json]: 👍

**Comment by yirmiyahu** [comment]
👍

**Pull request merged**